### ISSUE TITLE
Admin access and regular checks

### ISSUE THREAD

**Issue by proghead00** [benawad/vscode-stories#39]
It's sad to see that you had to disable uploading any further story. It's a great extension, and what you can do is, appoint admins for the same, and stories get uploaded only after a quick review by the admin.

**Comment by TheFern2** [comment]
@bar9 @Bukii  I think this is a good idea to appoint moderators, I don't think we're at that point but should definitely be on the nice to haves.
